docs/release/release_series.md

```diff
@@ -1,30 +1,40 @@
+DateReviewed: 2023-08-25
 title: Release Series
 
 Release Series
 ==============
 
-An OSG release series is a sequence of OSG software releases that are intended to provide a painless upgrade path.
-For example, the 3.2 release series contains OSG software 3.2.0, 3.2.1, 3.2.2, and so forth.
-A release series corresponds to a set of Yum software repositories, including ones for development, testing, and
-production use.
-The Yum repositories for one release series are completely distinct from the repositories for a different release
-series, even though they share many common packages.
-A particular release within a series is a snapshot of packages and their exact versions at one point in time.
-When you install software from a release series, say 3.2, you get the most current versions of software packages within
-that series, regardless of the current release version.
-
-When a new series is released, it is an opportunity for the OSG Technology area to add major new software packages, make
+OSG Software releases are organized into release series, with the intent that software updates within a series do will
+not take require manual configuration updates, cause significant downtime, or break dependent software.
+
+New series can be more disruptive, allowing OSG to add, substantially change, and remove software components.
+
+Support Policy
+--------------
+
+The OSG Software Team supports at most two concurrent release series, __current__ and __previous__, where the goal is to
+begin a new release series about every 12 months.
+Once a new series starts, the Software Team will support the previous series until the __next__ release series and will
+announce its end-of-life date at least 6 months in advance.
+
+When support ends for a release series, it means that the Software Team no longer updates the software, fixes issues, or
+troubleshoots installations for releases within the series.
+The plan is to maintain interoperability between supported series, but there is no guarantee that unsupported series
+will continue to function.
+
+Files for release series older than current or previous will be removed from the OSG Software repositories no earlier
+than when support ends for the previous release.
+For example, files for OSG 3.2 were not removed until May 2018, when support ended for OSG 3.3 in May 2018.
+
+When a new series is released, it is an opportunity for the OSG Software Team to add major new software packages, make
```

Comment:
Why move this down here?  This makes the line above

> New series can be more disruptive, allowing OSG to add, substantially change, and remove software components.

seem like a non sequitur.

Reply:
Yeah, I can move this paragraph back up. It is a bit redundant with the above